cli/internal/fs/turbo_json.go

```diff
@@ -174,14 +232,14 @@ type TaskOutputs struct {
 }
 
 // Sort contents of task outputs
-func (to TaskOutputs) Sort() TaskOutputs {
+func (to *TaskOutputs) Sort() *TaskOutputs {
```

Comment:
Can you talk about why this needs to a reference? It looks like we're immediately dereferencing it in `taskhash.go`

Reply:
It doesn't anymore. This is a remnant of a previous attempt where I hadn't separated the ResolvedTaskDefinition struct, so I needed `TaskOutputs` to be a pointer, and that caused downstream changes. Reverted